src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -50,15 +50,24 @@ export const TradeConfirm = ({ history }: RouterProps) => {
     state: { wallet, isConnected },
     dispatch,
   } = useWallet()
-  const { chain, tokenId } = trade.sellAsset
+  const { chain, chainId, tokenId } = trade.sellAsset
   const network = NetworkTypes.MAINNET
   const assetNamespace = AssetNamespace.ERC20
   const extra = tokenId
     ? { assetNamespace, assetReference: tokenId }
     : { assetNamespace: AssetNamespace.Slip44, assetReference: AssetReference.Ethereum }
-  const assetId = toAssetId({ chain, network, ...extra })
 
-  const status = useAppSelector(state => selectLastTxStatusByAssetId(state, assetId))
+
+  const accountSpecifier = useAppSelector(state =>
+    selectFirstAccountSpecifierByChainId(state, chainId),
+  )
+
+  const parsedTxId = useMemo(
+    // see makeUniqueTxId in txHistorySlice/utils
```

Comment:
Note: We could actually use `makeUniqueTxId` for that but it would require to change its signature/implementation to not support a Tx type arg, but rather individual parts, which would require a lot of changes across the codebase, and it doesn't make sense to clutter the diff with 25 more changes across 5/6 files for this IMO

Reply:
i really don't like this - the whole point of having centralized serialization/deserialization is to avoid this duplication of logic that gets out of sync.

Reply:
Implemented signature change in https://github.com/shapeshift/web/pull/1802